pull_request_template.md

```diff
@@ -1,38 +1,40 @@
-# Description 
-A summary of the change 
+# Description
 
+A summary of the change
+
+# Type of change
 
-# Type of change 
 - [ ] Bug fix
 - [ ] New feature
 - [ ] Refactoring
-- [ ] Documentation 
-
+- [ ] Documentation
+- [ ] Improvement
 
 # Problem
-Briefly describe the root cause and analysis of the problem 
 
+Briefly describe the root cause and analysis of the problem
```

Comment:
Any thoughts about whether we should remove this section because this is usually said in the description?

Reply:
+!